cmd/jwt-wallet/main_test.go

```diff
@@ -203,10 +233,14 @@ func TestIncorrectAddress(t *testing.T) {
 	assert.Equal(t, "address does not match public key", env.ClientRes.Body)
 }
 
-func GenerateClaims(addr string, pubKey *secp256k1.PublicKey) *signing.Claims {
+func GenerateClaims(addr string, pubKey *secp256k1.PublicKey, urlEncodeKey bool) *signing.Claims {
```

Comment:
Prefer passing in encoder or some typed object over booleans. The bool params make code harder to read and require delving into the function call to interpret meaning. Passing the `base64.StdEncoding` or `base64.URLEncoding` object is much clearer.

Reply:
oh, we're in test code here, nevermind!

Reply:
doesn't bother me either way--updated to pass the encoding